front-end/src/components/CreatePost.tsx

```diff
@@ -23,29 +24,51 @@ const CreatePostComponent = (props: any) => {
         }
     }
 
+    function parseCategories(categories: string) {
+        let parsed: string[] = categories.split(" ")
+        setCategories(parsed);
+    }
+
     function sendPost(e:any) {
+        e.preventDefault();
+
         let data = {
             title: title,
             description: desc,
             visibility: visibility,
             unlisted: unlisted,
             contentType: contentType,
             content: content,
-            categories: [categories]
+            categories: categories
         }
-        
+        console.log(data)
         axios.post(process.env.REACT_APP_API_URL + "/api/author/" + props.loggedInUser.authorId + "/posts/", data)
+        .then(res => {
+            if (res.status >= 400) {
+                setShowError(true)
+            } else if (res.status === 201) {
+                props.history.push("/")
+            }
+        }).catch(error => {
+            setShowError(true)
+        })
     }
 
     return (
         <div>
+            {showError ? <Alert>Could not create post</Alert> : null}
             <h1>Create Post</h1>
             <Form inline={true} onSubmit={e => sendPost(e)}>
                 <Input type="text" name="Title" placeholder="Title" onChange={e => setTitle(e.target.value)}/>
                 <Input type="text" name="Description" placeholder="Description" onChange={e => setDesc(e.target.value)}/>
-                <Input type="text" name="Content Type" placeholder="Content Type" onChange={e => setContentType(e.target.value)}/>
+                <Label for="Content Type">Content Type</Label>
+                <select name="Content Type" onChange={e => setContentType(e.target.value)}>
+                    <option value="Text">Text</option>
```

Comment:
Dropdown looks good!
I could have better clarified this, but according to one of Justins comments on the spec document
```
from the Post JSON defined in the spec, only the following are possible:

# text/markdown -- common mark
# text/plain -- UTF-8
# application/base64
# image/png;base64 # this is an embedded png -- images are POSTS. So you might have a user make 2 posts if a post includes an image!
# image/jpeg;base64 # this is an embedded jpeg
```
Could you make the values for the dropdown "image/jpeg;base64" and such? If need be we can change this later when porting it to a modal